calibration/CTDMOG/CGINS-CTDMOG-13589__20170403.csv

```diff
@@ -21,5 +21,5 @@ serial,name,value,notes
 37-13589,CC_ptcb2,0.00E+00,
 37-13589,CC_ptempa0,1.461928e+002,
 37-13589,CC_ptempa1,-7.005703e-002,
-37-13589,CC_ptempa2,5.947919e-007,
-37-13589,CC_wbotc,6.1595e-008,
+37-13589,CC_ptempa2,5.947920e-07,
+37-13589,CC_wbotc,6.159538e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT